src/tools/imgtool/main.cpp

```diff
@@ -106,15 +111,15 @@ static int parse_options(int argc, char *argv[], int minunnamed, int maxunnamed,
 	return lastunnamed;
 
 filternotfound:
-	fprintf(stderr, "%s: Unknown filter type\n", value);
+	std::wcerr << wstring_from_utf8(value) << L": Unknown filter type" << std::endl;
```

Comment:
Oh dear, this kind of change will break the possibility of every localising this.  Can you do something more like:

    util::stream_format(std::wcerr, L"%1$s: Unknown filter type\n", value);

I tested the stream format stuff with `wchar_t` when I was writing it, so it should work, but yell at me if it doesn't.  Same applies to all other examples of this.